test/conftest.py

```diff
@@ -238,7 +239,9 @@ def __init__(self, blob_host, blobs):
 
         @self.function_body
         def default_function_body(*args, **kwargs):
-            return sum(arg**2 for arg in args) + sum(value**2 for key, value in kwargs.items())
+            return sum(arg**2 for arg in args if isinstance(arg, numbers.Number)) + sum(
+                value**2 for key, value in kwargs.items() if isinstance(value, numbers.Number)
+            )
```

Comment:
This change is needed because grouped functions would pass in a `modal_q` kwarg that is a modal Queue rather than a number, which would cause issues.